index.html

```diff
@@ -43,10 +49,11 @@ <h1 class="about__title">I’m Mariane <br> Glad to see <br> you!</h1>
         </ul>
     </div>
   </section>
-  <section class="works">
+  <section class="works" id="myWorks">
     <div class="works__item tonic__project">
         <div class="tonic__header"></div>
-        <h3 class="project__title">Tonic</h3>
+        <div class="tonic__details">
+            <h3 class="project__title">Tonic</h3>
         <div class="project__tags">
             <p>CANOPY</p>
             <div class="project__tags-boulet gray_color"></div>
```

Comment:
- [ ] I see that the height of your project cards is too small compared to the Figma Design. Please fix this

Figma Design: height `496px` 
![image](https://user-images.githubusercontent.com/84156714/220799031-0cf258a2-2440-4917-8f78-add6ad531095.png)

Your design: height `272px`
![image](https://user-images.githubusercontent.com/84156714/220799083-7bdce573-02ab-486f-b151-1485e0c37b46.png)

Reply:
✅ Done